projects/ngrx.io/content/marketing/enterprise-support.md

```diff
@@ -0,0 +1,74 @@
+
+Additionally, the following topics can be covered for your teams:
+
+- How to set up Frontend teams for success with proper development processes.
+- Testing Angular applications. Unit and end-to-end tests.
+
+## Participating NgRx team members
+
+<div class="team-grid">
+    <ngrxio-contributor json='{"name": "Mike Ryan", "picture": "mike-ryan.jpg"}'></ngrxio-contributor>
+    <div class="member-description">Mike Ryan is a Principal Architect at LiveLoveApp and one of the original
+        co-creators of NgRx and a Google Developer Expert in Angular.
+        <p>He wrote the initial implementations for <code>@ngrx/effects</code>, <code>@ngrx/store-devtools</code>, and
+            <code>@ngrx/entity</code>.</p>
+        <p>Since creating NgRx, Mike has helped small startups and large enterprises (including Google!) successfully
+            implement reactive Angular applications using NgRx.</p>
+    </div>
+    <ngrxio-contributor json='{"name": "Alex Okrushko", "picture": "alex-okrushko.jpg"}'></ngrxio-contributor>
+    <div class="member-description">Alex Okrushko is a Principal Architect at Cisco CX - Customer Experience. He is part
+        of the NgRx team, Google Developer Expert in Angular, @AngularToronto organizer and co-organizer of the official
+        Angular Discord.
+        <p>Alex has been contributing to NgRx since 2018. Among his contributions are creator factories (such as
+            <code>createAction()</code> and <code>createReducer()</code>), overall type strictness of the NgRx code and
+            introduction of <code>@ngrx/component-store</code> library.</p>
+        <p>Prior to working at Cisco, Alex worked over 7 years at Google in a few teams, including Firebase.</p>
+        <p>In his free time, he loves to learn & share the knowledge, provides NgRx workshops and helps with <a
+                href="https://ts.dev/style">ts.dev/style</a> - the popular TypeScript style guide.</p>
+    </div>
+    <ngrxio-contributor json='{"name": "Marko Stanimirović", "picture":"marko.jpg"}'></ngrxio-contributor>
+    <div class="member-description">Marko Stanimirović is a core member of the NgRx team (contributing since 2020),
+        a Google Developer Expert in Angular, and an organizer of the Angular Belgrade group.
+        <p>Marko's contributions include <code>createFeature()</code>, overhaul of the <code>@ngrx/component</code>
+            library, and continuous maintenance of NgRx libraries.</p>
+        <p>He enjoys contributing to open source software, sharing knowledge through technical articles and talks, and
+            playing guitar.</p>
+        <p>Marko is also a Master of Science in Software Engineering from the University of Belgrade.</p>
+    </div>
+</div>
+
+## Contact
+
+Get in touch with the team to see how we can make you and your project successful. Let us know which services you are interested in.
+
+<ngrx-contact-form></ngrx-contact-form>
```

Comment:
These changes should make it more engaging.

```suggestion
<h1 class="banner-headline">Enterprise-level Support from NgRx team members</h1>

NgRx collection of libraries continues to stay open-sourced MIT-licensed and free to use. At the same time, we hear from companies that designing high-quality architecture could be challenging and additional help and guidance are needed.

Take your Angular app to a new level with **premium services** from [participating NgRx team members](#participating-ngrx-team-members).

[Send us a message.](#contact)

## Provided services
- [**NgRx support**](#ngrx-support)
- [**Angular/NgRx consulting**](#angularngrx-consulting)
- [**Trainings**](#trainings)

### NgRx Support
Does your team need period help with general questions related to NgRx, Angular, and RxJS in a timely manner?

Is there a particular NgRx feature or a bug that your team would like to get prioritized?

Have you encountered an issue? With **NgRx support**, our team can assist either through email or Video Call in a reasonable time frame.

### Angular/NgRx consulting
When your teams need a more hands-on approach with the architecture and implementation of your Angular/NgRx app that follows best practices from a team of experts, then this is the right choice for your company.

Whether your team is starting with a green field project, is introducing state management into the app, or has been using NgRx for a while, we can help to ensure that the architecture is set up for scalability and performance while improving the app's quality.

### Trainings
Two or three full days of **NgRx workshop** that covers from the basics of NgRx to the most advanced topics. Whether your teams are just starting with NgRx or have been using it for a while - they are guaranteed to learn new concepts during this training.

The workshop covers both NgRx global `store` and `ComponentStore`.

Additionally, the following topics can be covered for your teams:

- How to set up Frontend teams for success with proper development processes.
- Testing Angular applications. Unit and end-to-end tests.

## Participating NgRx team members

<div class="team-grid">
    <ngrxio-contributor json='{"name": "Mike Ryan", "picture": "mike-ryan.jpg"}'></ngrxio-contributor>
    <div class="member-description">Mike Ryan is a Principal Architect at LiveLoveApp and one of the original
        co-creators of NgRx and a Google Developer Expert in Angular.
        <p>He wrote the initial implementations for <code>@ngrx/effects</code>, <code>@ngrx/store-devtools</code>, and
            <code>@ngrx/entity</code>.</p>
        <p>Since creating NgRx, Mike has helped small startups and large enterprises (including Google!) successfully
            implement reactive Angular applications using NgRx.</p>
    </div>
    <ngrxio-contributor json='{"name": "Alex Okrushko", "picture": "alex-okrushko.jpg"}'></ngrxio-contributor>
    <div class="member-description">Alex Okrushko is a Principal Architect at Cisco CX - Customer Experience. He is part
        of the NgRx team, Google Developer Expert in Angular, @AngularToronto organizer and co-organizer of the official
        Angular Discord.
        <p>Alex has been contributing to NgRx since 2018. Among his contributions are creator factories (such as
            <code>createAction()</code> and <code>createReducer()</code>), overall type strictness of the NgRx code and
            introduction of <code>@ngrx/component-store</code> library.</p>
        <p>Prior to working at Cisco, Alex worked over 7 years at Google in a few teams, including Firebase.</p>
        <p>In his free time, he loves to learn & share the knowledge, provides NgRx workshops and helps with <a
                href="https://ts.dev/style">ts.dev/style</a> - the popular TypeScript style guide.</p>
    </div>
    <ngrxio-contributor json='{"name": "Marko Stanimirović", "picture":"marko.jpg"}'></ngrxio-contributor>
    <div class="member-description">Marko Stanimirović is a core member of the NgRx team (contributing since 2020),
        a Google Developer Expert in Angular, and an organizer of the Angular Belgrade group.
        <p>Marko's contributions include <code>createFeature()</code>, overhaul of the <code>@ngrx/component</code>
            library, and continuous maintenance of NgRx libraries.</p>
        <p>He enjoys contributing to open source software, sharing knowledge through technical articles and talks, and
            playing guitar.</p>
        <p>Marko is also a Master of Science in Software Engineering from the University of Belgrade.</p>
    </div>
</div>

## Contact

Get in touch with the team to see how we can make you and your project successful. Let us know which services you are interested in.

<ngrx-contact-form></ngrx-contact-form>
```